index.js

```diff
@@ -47,15 +47,11 @@ function serverTiming(options) {
      * Add server timing to response headers
      */
     onHeaders(res, function addHeader() {
-      if (this.getHeader(SERVER_TIMING_HEADER)) {
-        console.warn('Server timing header already exists, aborted')
-        return
-      }
-
-      const headerString = Object.keys(headers).map(key => {
+      var existingHeaders = this.getHeader(SERVER_TIMING_HEADER)
+      const headerString = (existingHeaders || []).concat(Object.keys(headers).map(key => {
```

Comment:
Good job!

Seems that `existingHeaders` is a string, should it be `existingHeaders ? [existingHeaders] : []`?